src/components/todo/Todo.js

```diff
@@ -12,8 +20,46 @@ class Todo extends Component {
     };
   }
 
+  onClick = (index) => {
+    console.log(this.state.todoList)
+    let newTodoList = Array.from(this.state.todoList)
+    newTodoList.splice(index, 1);
+    this.setState({ todoList: newTodoList });
+  }
+
+  getTodo = () => {
+    if (this.state.todoList.length === 0) {
+      return "표시할 TODO가 없어요!";
+    }
+    else {
+      let showTodoList = [];
+      for (let index = 0; index < this.state.todoList.length; index++) {
+        showTodoList.push(
+          <TodoCard
+            index={index}
+            todo={this.state.todoList[index]}
+            onClick={this.onClick}
+          />);
+      }
+      return showTodoList;
+    }
+  }
+
+  onSubmit = (_todo) => {
+    let newTodoList = Array.from(this.state.todoList);
+    newTodoList.push(_todo);
+    this.setState({
+      todoList: newTodoList
```

Comment:
```suggestion
      todoList: [...this.state.todoList, _todo]
```
spread operator를 이용해 깔끔하게 리팩토링할 수 있습니다~

Reply:
훨씬 깔끔한 느낌이네요! 감사합니다